SOCI/README.md

```diff
@@ -0,0 +1,437 @@
+# SOCI Index for AWS Deep Learning Containers
+
+## Background on SOCI Index
+
+The Software Component Index (SOCI) is a technology that enables efficient container image management through selective file downloading. It uses a layer-based indexing system to map file locations within container images, allowing containers to start with only the necessary files loaded (lazy loading). This approach significantly reduces network bandwidth usage and improves container startup times, making it particularly valuable for organizations managing large container images in cloud environments.
+
+A SOCI index, which enables lazy loading of container images, consists of two key components:
+
+1. **SOCI Index Manifest**: Lists zTOCs and references the target image
+2. **zTOCs (Zone Table of Contents)**: Comprising:
+   - **TOC**: A table of contents containing file metadata and corresponding offset information in the decompressed TAR archive
+   - **zInfo**: A collection of checkpoints representing compression engine states at various points in the layer
+
+## SOCI Index Manifest V1 vs V2
```

Comment:
Do we need to callout the difference between v1 vs v2? Since we started support SOCI index starting with v2, can we just say that we uses SOCI inde manifest v2 and link specific SOCI documentation instead?